src/usr/local/buildpack/utils/cache.sh

```diff
@@ -37,8 +40,9 @@ function download_file () {
   name=${2:-$(basename "${url}")}
 
   local temp_folder=${BUILDPACK_CACHE_DIR:-${TEMP_DIR}}
-  curl --create-dirs -sSfLo "${temp_folder}/${name}" "${url}"
-  echo "${temp_folder}/${name}"
+  if curl --create-dirs -sSfLo "${temp_folder}/${name}" "${url}" >/dev/null 2>&1 ; then
+    echo "${temp_folder}/${name}"
+  fi
```

Comment:
Didn't we aggreed download should fail all? 🤔

Reply:
we did, but I haven't changed the line yet 🙈